ZeeguuAPI/ZeeguuAPI/ZeeguuAPIDefaults.swift

```diff
@@ -84,23 +108,33 @@ extension ZeeguuAPI {
 	
 	func sendAsynchronousRequest(request: NSURLRequest, completion: (response: String?, error: NSError?) -> Void) {
 		let session = NSURLSession.sharedSession()
-		NSLog("Sending request for url \"%@\": %@\n\n", request.URL!, request);
+		if (self.enableDubugOutput) {
```

Comment:
Actually, that was inside the enableDebug guard, you can see in the diff that the line was removed and was added within the guard. I did make a debugPrint function however.